### PR TITLE
chore(docs): clarify on usage of packages via monorepo

### PR DESCRIPTION
**Pull request by achrinza**
closes: #4224

Signed-off-by: Rifa Achrinza <25147899+achrinza@users.noreply.github.com>

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: sandbox/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ commands:
 ```sh
 cd loopback-next
 npm install
+npm build
```
Comment (agnes512) [inline sandbox/README.md]: `npm run build`?

Comment (agnes512) [inline sandbox/README.md]: `npm run build`?

Comment (achrinza) [reply sandbox/README.md]: Whoops, nice catch!

File: docs/site/DEVELOPING.md [hunk 2]
```diff
@@ -80,6 +81,25 @@ Please note that we are automatically running the build from `pretest` script,
...
+### Using monorepo packages as dependencies
+
+The `/sandbox` directory in the monorepo can be used to utilize the source code
+as symbolically-linked dependencies. See the
+[README](https://github.com/strongloop/loopback-next/blob/master/sandbox/README.md)
+for usage instructions.
+
+### Using the CLI via the monorepo
+
+The CLI can be executed from `/loopback-next/packages/cli/bin/cli-main.js`.
```
Comment (nabdelgadir) [inline docs/site/DEVELOPING.md]: Can you add a direct link to this file? And show how it can be executed?

Comment (raymondfeng) [reply docs/site/DEVELOPING.md]: Suggestion:

The CLI can be invoked from the local `loopback-next` git repo: 

```sh
./loopback-next/packages/cli/bin/cli-main.js
```

The arguments are the same as `lb4`, for example, `./loopback-next/packages/cli/bin/cli-main.js model` is the same as `lb4 model`.

File: docs/site/DEVELOPING.md [hunk 3]
```diff
@@ -80,6 +81,25 @@ Please note that we are automatically running the build from `pretest` script,
 therefore you should not need to run this command as part of your
 [red-green-refactor cycle](http://www.jamesshore.com/Blog/Red-Green-Refactor.html).
 
+## Utilizing the monorepo
+
+The monorepo's source code can be utilized for testing new features or bug fixes
+withing LoopBack itself or applications are dependent on LoopBack.
```
Comment (raymondfeng) [inline docs/site/DEVELOPING.md]: Suggestion:
Source code from the monorepo can be utilized as LoopBack 4 packages that are not published to npm registry yet. This is useful to test or debug your applications or extensions with project dependencies on LoopBack 4 against a local git repo and branch of `loopback-next`.

**Comment by achrinza** [comment]
@nabdelgadir @raymondfeng updated as per-your comments

**Pull request merged**